components/global/footer/Footer.tsx

```diff
@@ -1,127 +1,56 @@
+              <li>
+                <Link
+                  href="https://ereolengo.dk/inspiration/til-skoler"
+                  className="animate-text-underline">
+                  Skoler og UNI-login
+                </Link>
+              </li>
+              <li>
+                <Link
+                  href="https://ereolengo.dk/content/bliv-boganbefaler"
+                  className="animate-text-underline">
+                  Bliv boganbefaler
+                </Link>
+              </li>
             </ul>
           </div>
-
-          <div>
-            <h2 className="mb-4 text-xl font-semibold">Kontakt</h2>
-            <p className="mb-4">
-              Lyngby-Taarbæk Bibliotekerne
-              <br />
-              Lyngby Hovedgade 28
-              <br />
-              2800 Kgs. Lyngby
+          <div className="mt-10 flex flex-col items-center justify-end">
+            <div className="bg-foreground mb-10 ml-0 flex flex-0 items-center rounded-full lg:ml-auto">
+              <Icon name="logo-with-outline" className="text-background h-auto w-52" />
+            </div>
+            <p className="text-typo-subtitle-lg ml-auto">
+              En del af Bibliotek og Borgerservice Nordfyns Kommune
             </p>
-            <p className="mb-4">
-              +45 45 97 37 00
-              <br />
-              Man - fre / Kl. 10-16
-            </p>
-            <Link href="mailto:info@ltk.dk" className="underline">
-              info@ltk.dk
-            </Link>
-          </div>
-        </div>
-
-        <div className="border-border mt-8 flex justify-between border-b pb-8">
-          <select className="rounded border px-2 py-1">
-            <option>DK</option>
-          </select>
-          <Icon className="h-[40px]" name="library-logo" />
-        </div>
-
-        <div className="mt-8 flex flex-col items-center justify-between md:flex-row">
-          <div className="mb-4 flex items-center space-x-4 md:mb-0">
-            <Link href="#" className="text-gray-600 hover:text-gray-900">
-              <Facebook size={20} />
-              <span className="sr-only">Facebook</span>
-            </Link>
-            <Link href="#" className="text-gray-600 hover:text-gray-900">
-              <Instagram size={20} />
-              <span className="sr-only">Instagram</span>
-            </Link>
-          </div>
-          <div className="flex flex-wrap justify-center space-x-4 md:justify-end">
-            <Link href="#" className="hover:underline">
-              Behandling af persondata
-            </Link>
-            <Link href="#" className="hover:underline">
-              Servicedeklaration
-            </Link>
-            <Link href="#" className="hover:underline">
-              Relement
-            </Link>
-            <Link href="#" className="hover:underline">
-              Tilgængelighed
-            </Link>
           </div>
         </div>
+        <hr className="go-divider mt-10" />
```

Comment:
`go-divider` is not a TW class you can use anymore. the styles is applied on the `<hr>`

```suggestion
        <hr className="mt-10" />
```

Its also still present on the `ResponsiveDialog` component. Can you remove the class from there also while you're at it?